contrib/tracing/log_mempool_activity.bt

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bpftrace
+
+BEGIN
+
+{
+  printf("Logging Mempool Activity\n");
+  printf("  REASON ,  TIME  ,EVENT,    AMT SATS    ,                            TXID                                ,   SIZE  ,   FEE SATS\r\n");
+}
+
+usdt:./src/bitcoind:mempool:event
+{
+    //64bit chunks of the 256bit TXID
+    $p1 = (uint64)0;
+    $p2 = (uint64)0;
+    $p3 = (uint64)0;
+    $p4 = (uint64)0;
+    //Reason
+    $reason = (int32)arg3;
+    $reasonStr = "DEFAULT";
+    //TX Size
+    $size = (uint32) arg4;
+    //Args
+    $amt = (uint64) arg1;
+    $txid = arg2;
+    $event = str(arg0);
+    //Pointer Ref
+    $p = $txid + 24;
+    //String Decoding
+    if($reason == 5){
+        $reasonStr = "REPLACED"
+    }
+    if($reason == 4){
+        $reasonStr = "CONFLICT"
+    }
+    if($reason == 3){
+        $reasonStr = "BLOCK"
+    }
+    if($reason == 2){
+        $reasonStr = "REORG"
+    }
+    if($reason == 1){
+        $reasonStr = "SIZELIMIT"
+    }
+    if($reason == 0){
+        $reasonStr = "EXPIRY"
+    }
+    //Store the segments
+    unroll(4){
```

Comment:
The above unroll loop causes the printf statement to execute on each iteration. This causes dropped traced events that show up as visual artifacts or make for an usable output file. To get a perfect output there should be one printf statement per trace event. 

This PRs bpftrace script may not be the best looking but it was implemented so that under the default bpftrace env. The output data will uncompromised and always product a usable result.